README.md

```diff
@@ -16,6 +16,8 @@ $ make install
 ### Init-heimdall 
 ```bash 
 $ heimdalld init
+$ heimdalld init --chain=mainnet        Will init with genesis.json for mainnet
```

Comment:
Is this still need it? I though we removed this and was integrated inside the daemon command.

Reply:
Yes, init will be executed through start command if genesis.json file does not exist. But i leave init command as it was before (for example if someone wants to init everything again for different network)